src/__tests__/http-test.ts

```diff
@@ -28,14 +26,14 @@ import {
 
 import { graphqlHTTP } from '../index';
 
-// TODO Improve typings after converting to TypeScript
-type Server = () => {|
-  get: (...args: Array<mixed>) => mixed,
-  post: (...args: Array<mixed>) => mixed,
-  put: (...args: Array<mixed>) => mixed,
-  request: () => any,
-  use: (...args: Array<mixed>) => any,
-|};
+type Middleware = (req: any, res: any, next: () => void) => any;
+type Server = () => {
+  request: () => supertest.SuperTest<supertest.Test>;
+  use: (middleware: Middleware) => any;
+  get: (path: string, middleware: Middleware) => any;
+  post: (path: string, middleware: Middleware) => any;
+  put: (path: string, middleware: Middleware) => any;
```

Comment:
@danielrearden Do we use return values anywhere?
If not it can be changed to unknown.
I just grep from time to time for `any` to remove workarounds inside typings so would like to minimize the usage of `any` in actual code.